.github/workflows/release.yml

```
@@ -0,0 +1,195 @@
+
+          ### Compatibility
+          - Requires Ignition 8.3.1 or later
+          - Requires Java 17+
+
+          For version 8.1.x support, see [v1.0.3-ignition-8.1](https://github.com/WHK01/ignition-git-module/releases/tag/v1.0.3-ignition-8.1)
+          EOF
+          fi
+
+      - name: Create GitHub Release
+        if: steps.version.outputs.is_release == 'true'
+        uses: softprops/action-gh-release@v1
```

actionlint (1.7.9)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)